### ISSUE TITLE
复现

### ISSUE THREAD

**Issue by 139352** [ICML2020-submission/MAGCN#3]
你好，请问有什么具体的参数要求吗，为什么我复现的精度是最好只有百分之六十多

**Comment by sxu-yaokx** [comment]
Maybe you should change the "label_rate" parameter (29-30 lines in "train.py"), which is a parameter used in the robustness analysis in experiments (see Fig. 4 and Fig. 5). For a normal MAGCN, the label_rate should be set to 1.

**Comment by 139352** [comment]
Thank you very much for your answer! After I changed the label-rate you said to 1, the code will always report an error list index out of range on line 67 of utils, and in the division of the verification set and the training set, idx_val = range(len(y), len(y )+500) is the same error, I looked at the description of the paper again, the ablation experiment said that we changed the attack ratio (the ratio of deleted edges) from 0.1 to 1, and reported the experimental results in Figure 4, whether Does it mean that the normal value of rate is 0.1? The result after I changed it to 0.1 is similar to the result in your paper.

**Comment by 139352** [comment]
作者您好，我之前在github上面问过您复现的问题，您当时告诉我说让我更改一下label- rate，我之前论文读得不够透彻，今天突然发现一个问题，回去看了一下您的代码，那个label- rate，应该是数据集的划分标准，如果跟GAT的Cora数据集83.0的表现对比的话，数据集的训练集节点是20个，正好和你label- rate的0.01对上，那么应该是不能通过更改这个值去提高表现能力的，我不知道我理解的是否有误，这对我非常重要，很希望您可以回复我一下，非常感谢您



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Kaixuan Yao ***@***.***&gt;
发送时间: 2023年8月9日 20:59
收件人: sxu-yaokx/MAGCN ***@***.***&gt;
抄送: 139352 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [sxu-yaokx/MAGCN] 复现 (Issue #3)





 
Maybe you should change the "label_rate" parameter (29-30 lines in "train.py"), which is a parameter used in the robustness analysis in experiments (see Fig. 4 and Fig. 5). For a normal MAGCN, the label_rate should be set to 1.

**Comment by Enjoy0223** [comment]
您好，我想请问一下，为什么我试图运行的时候显示load_data()只需要一个参数，而源代码里load_data(FLAGS.dataset，label_rate)却试图传两个参数
